webfonts/_red-hat-font.scss

```diff
@@ -9,25 +9,30 @@
   $style: "normal",
   $relative: true
   ) {
-    
+
   $filePath: "./" + $familyName + "/" + $familyName + "-" + $weightName;
- 
+
   @if $relative == false {
     $filePath: $fontLocation +  "/" + $familyName +  "/" + $familyName + "-" + $weightName;
   }
- 
+
+  // Using woff & woff2 gives us the following browser support:
+  // Chrome 5+
+  // Safari 5.1+
+  // Firefox 3.6+
+  // Opera 11.50+
+  // IE 9+
+  // Android 4.4+
+  // iOS 5.1+
   @font-face {
     font-family: $familyName;
-    src: 
-      url('#{$filePath}.eot'); /* IE9 Compat Modes */
```

Comment:
ah, if we are dropping the EOT reference / ie8, then should this be a 3.0.0? @zhawkins

Reply:
Yes that makes sense. It would be a breaking change for IE8 users 🤔. I'll make it so.